.vscode/settings.json

```diff
@@ -1,6 +1,7 @@
 // Place your settings in this file to overwrite default and user settings.
 {
   "eslint.enable": true,
+  "deno.enable": false,
```

Comment:
Please remove this added line.

Reply:
Is SQLTools using Deno? For those with the Deno VScode extensions, development is hampered by the Deno extension thinking this is a Deno project and effectively removing all typing checks and imports. 😕

Reply:
No, SQLTools isn't using Deno AFAIK. I think it's better not to have settings in the project which relate to extensions we don't require. Instead I suggest you open the SQLTools workspace from its .code-workspace file, then go to your Extensions view, locate your Deno extension, click on its gearwheel and choose to disable it for the workspace. You might be asked to reload the workspace, but thereafter Deno shouldn't interfere with it even after you remove that extra line from /vscode/settings.json